plugins/woocommerce/client/admin/client/settings-payments/components/ellipsis-menu-content/ellipsis-menu-content.tsx

```diff
@@ -18,17 +18,49 @@ import { useState } from '@wordpress/element';
 import './ellipsis-menu-content.scss';
 
 interface EllipsisMenuContentProps {
+	/**
+	 * The ID of the payment provider.
+	 */
 	providerId: string;
+	/**
+	 * The file path of the plugin associated with the payment gateway.
+	 */
 	pluginFile: string;
+	/**
+	 * Indicates if the menu is being used for a payment extension suggestion.
+	 */
 	isSuggestion: boolean;
+	/**
+	 * The URL to call when hiding a payment extension suggestion. Optional.
+	 */
 	suggestionHideUrl?: string;
+	/**
+	 * Callback to close the ellipsis menu.
+	 */
 	onToggle: () => void;
+	/**
+	 * Array of links related to the payment provider.
+	 */
 	links?: PaymentGatewayLink[];
+	/**
+	 * Indicates if the account can be reset. Optional.
+	 */
 	canResetAccount?: boolean;
+	/**
+	 * Callback to show or hide the reset account modal. Optional.
+	 */
 	setResetAccountModalVisible?: ( isVisible: boolean ) => void;
+	/**
+	 * Indicates if the payment gateway is enabled. Optional.
```

Comment:
I would be more specific: "Indicates if the payment gateway is enabled for payment processing. Optional."

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/1cb4f8eb9290355d38f8cefebb9784d70bdf6450